dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/oracle/OracleMetadataConnector.java

```diff
@@ -126,6 +126,9 @@ public void addTasksTo(@Nonnull List<? super Task<?>> out, @Nonnull ConnectorArg
     String ownerInList = toInList(arguments.getSchemata());
     String whereCondOwner = ownerInList == null ? "" : " WHERE OWNER IN " + ownerInList;
     String whereCondTableOwner = ownerInList == null ? "" : " WHERE TABLE_OWNER IN " + ownerInList;
+    String whereCondSequenceNotSystemGeneratedAndOwner =
+        ownerInList == null ? " WHERE SEQUENCE_NAME NOT LIKE 'ISEQ$$\\_%' ESCAPE '\\'"
+            : " WHERE SEQUENCE_NAME NOT LIKE 'ISEQ$$\\_%' ESCAPE '\\' AND SEQUENCE_OWNER IN " + ownerInList;
     String whereCondSequenceOwner =
         ownerInList == null ? "" : " WHERE SEQUENCE_OWNER IN " + ownerInList;
```

Comment:
Seems to me this might be computed for nothing now?

Reply:
Can you please clarify? `whereCondSequenceNotSystemGeneratedAndOwner` appears to be used around l305, or am I missing something?

Reply:
`whereCondSequenceOwner` appears unused.

Reply:
See around L205:
```
    buildSelectStarTask(
        out,
        Sequences.ZIP_ENTRY_NAME_DBA,
        "DBA_Sequences",
        Sequences.ZIP_ENTRY_NAME_ALL,
        "ALL_Sequences",
        whereCondSequenceOwner);
```

`SELECT *` tasks are fine with no extra filtering, only the task with `DBMS_METADATA.GET_XML` cannot handle newer system-generated sequences.

Reply:
I will close this conversation for now, feel free to re-open if you still see an issue and I missed something.